packages/components/psammead-grid/package-lock.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "psammead-grid",
+  "version": "1.0.0-alpha.0",
+  "lockfileVersion": 1,
+  "requires": true,
+  "dependencies": {
+    "@bbc/gel-foundations": {
+      "version": "3.4.0",
+      "resolved": "https://registry.npmjs.org/@bbc/gel-foundations/-/gel-foundations-3.4.0.tgz",
+      "integrity": "sha512-WuewciWzvSb/WKhUGSvIoighqCdXGN2PsVQbZbUo5/eWi5+o3O8uoojUpeQeRZMYWLsLVELPHr6qJDuqKghXKA=="
+    },
+    "@bbc/psammead-paragraph": {
```

Comment:
should be dev dependency

Reply:
Has been removed as a dependency in the package.json since it is a devDependency at the repo level (can be used in tests and in Storybook).